infra/ansible/roles/knot-authoritative/templates/knot.conf.j2

```diff
@@ -25,10 +25,23 @@ template:
     global-module: mod-stats
     storage: /var/lib/knot/zones
 
+key:
+  - id: doh.mesh.nycmesh.net
+    algorithm: hmac-sha384
+    secret: {{ TSIG_KEY_DOH }}
+
+acl:
+  - id: acl-update-doh
+    action: update
+    key: doh.mesh.nycmesh.net
```

Comment:
Restrict to `acme-challenge.doh.mesh.nycmesh.net. txt`?

Reply:
Good call. Limited to a specific txt record,